Don't include log.cpp in log.h. Besides being incredibly weird, it
causes build failure:
error: reference to 'log' is ambiguous
--- lastfmlib/utils/log.h.orig	2009-07-03 10:26:53.000000000 -0500
+++ lastfmlib/utils/log.h	2021-12-28 21:53:04.000000000 -0600
@@ -85,6 +85,5 @@
     void debug(const T1& t1, const T2& t2, const T3& t3, const T4& t4, const T5& t5, const T6& t6);
 }
 
-#include "log.cpp"
 
 #endif
